db/migrate/20220107173125_create_user_identity.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUserIdentity < ActiveRecord::Migration[6.1]
+  def change
+    create_table :user_identities, id: :uuid, force: :cascade do |t|
```

Comment:
Bonus: You could add 
```ruby
config.generators do |g|
  g.orm :active_record, primary_key_type: :uuid
end
```
to `config/application.rb` and make UUIDs the default primary key type. :)

Reply:
Thanks for pointing that out! I created a new PR for that change: #590